jekyll-assets/scripts/copy-to-clipboard.js

```diff
@@ -48,7 +48,24 @@ window.addEventListener('load', function() {
       if (trigger.parentNode.querySelector('div.line')) {
         var text = extractDoxygenCode(trigger.parentNode);
       } else {
-        var text = trigger.parentNode.querySelector('pre').textContent;
+        var text = trigger.parentNode.querySelector('pre').textContent
+        
+        // if the code snippet represents a console snippet
+        if (trigger.parentNode.querySelector('pre').querySelector('code') != null && trigger.parentNode.querySelector('pre').querySelector('code').getAttribute('data-lang') == 'console') {
+          // for each line of the code snippet
+          var text_split_into_lines = text.split('\n');
+
+          // trim the '$ ' from the clipboard copy so users don't have to trim it themselves
+          text_split_into_lines = text_split_into_lines.map(function(x) {
+            if (x.startsWith('$ ')) {
```

Comment:
@nathan-contino Will the `$` always be followed by a simple space, or do we need to check for any space char, e.g., tabs, etc... ?